PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,20 @@
 ## Change Summary
 
-> Description here
+> Link to Jira ticket if applicable
+> Describe the changes (even if there is a description in Jira)
+> Identify backwards incompatible changes; link to tickets/PRs that must merge or deploy first
+> Call out known issues, high risk areas, or future work to be done
+> Include guidance for reviewers if relevant
 
-## Type of change
```

Comment:
I don't have a specific thought for how it should happen, but it would be great to nudge engineers into confirming that they have made any documentation updates that reflect the new state of the project post-merge.

Reply:
I agree that documentation is important and sometimes overlooked. But also, only a tiny fraction of PRs require documentation updates. I'd rather not add what is essentially a checklist item that will rarely be checked, especially because it wouldn't add any insight for reviewers.

Instead, we could propose separate guidelines for what reviewers should look for in PRs. Here is an [example](https://docs.google.com/document/d/1026Xpf1Bp5zP3Kn3m4DtE3j-17wuh5KJfVM9E6osmdQ/edit?usp=sharing) I have used in the past.

Reply:
That makes sense and I agree that I don't think this needs to be a checklist item. I'm happy to have this be something that we can guide reviewers on. Maybe most reviewers will be able to tell when a PR includes changes worthy of a documentation update (I'm thinking mainly of README updates to things like instructions to run locally, testing, etc.